qemu/tests/cfg/numa_negative.cfg

```diff
@@ -0,0 +1,39 @@
+- numa_negative:
+    type = numa_negative
+    kill_vm = yes
+    kill_vm_gracefully = no
+    backend_mem = memory-backend-ram
+    use_mem = no
+    start_vm = no
+    smp = 2
+    vcpu_maxcpus = ${smp}
+    mem = 2048M
+    mem_devs = 'mem0 mem1'
+    size_mem = 1024M
+    guest_numa_nodes = 'node0 node1'
+    numa_memdev_node0 = mem-mem0
+    numa_memdev_node1 = mem-mem1
+    numa_cpus_node0 = '0'
```

Comment:
I guess we don't need quotation marks here?

Reply:
> I guess we don't need quotation marks here?

Yes, the quotation marks is not must, i am ok to remove it, thanks ~

Reply:
remove the non-must quotation marks, could you help review again, thanks!